src/librustc_mir/lib.rs

```diff
@@ -15,7 +15,7 @@ Rust MIR: a lowered representation of Rust. Also: an experiment!
 */
 
 #![cfg_attr(not(stage0), feature(nll))]
-#![cfg_attr(not(stage0), feature(infer_outlives_requirements))]
+#![cfg_attr(stage0, feature(infer_outlives_requirements))]
```

Comment:
Do we need this, really? It seems like we can't be relying on it...

Reply:
done

Reply:
although https://forge.rust-lang.org/stabilization-guide.html says that we want to leave it to compile the stage0 compiler.